package.json

```diff
@@ -29,10 +29,10 @@
   "license": "MIT",
   "dependencies": {
     "@types/react": "15.0.38",
+    "adaptivecards": "^1.0.0-beta2",
```

Comment:
Should this be non-carat so that it locks to the exact version?

Reply:
fixed 👍